services/movie.py

```diff
@@ -0,0 +1,45 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(
+        genres_ids: list[int] = None,
+        actors_ids: list[int] = None
+) -> QuerySet:
+    queryset = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        return queryset.filter(
+            genres__id__in=genres_ids,
+            actors__id__in=actors_ids
+        )
+    if genres_ids:
+        return queryset.filter(genres__id__in=genres_ids)
+    if actors_ids:
+        return queryset.filter(actors__id__in=actors_ids)
+
+    return queryset
+
+
+def get_movie_by_id(movie_id: int) -> Movie:
+    return Movie.objects.get(id=movie_id)
```

Comment:
Consider handling the case where a movie with the given ID does not exist. You can use `Movie.objects.get(id=movie_id)` inside a try-except block to catch `Movie.DoesNotExist` and handle it appropriately, or use `Movie.objects.filter(id=movie_id).first()` to return `None` if not found.